src/soap.js

```diff
@@ -565,6 +590,8 @@ class SoapMethodCall {
         const that = this;
         const promise = this._transport(this.request);
         return promise.then(function(body) {
+            if (body.indexOf(`XSV-350008`) != -1)
```

Comment:
Isn't there a better way to test for the error code than parsing the whole response body? Why not look at the SOAP fault ?